backend/dal/Migrations/Initial/PostDeploy/01-ProjectStatus.sql

```diff
@@ -65,21 +65,21 @@ INSERT INTO dbo.[ProjectStatus] (
 ), (
     6
     , 5
-    , 'Submitted'
-    , 'SU'
+    , 'Review'
+    , 'RE'
     , 1
     , 0
     , 'The project has been submitted for review to be added to the Surplus Property Program.'
-    , '/projects/submitted'
+    , '/projects/review'
     , 1
 ), (
     7
     , 6
-    , 'Property Review'
+    , 'Submitted'
     , 'AS-I'
     , 1
     , 0
-    , 'Property information review.'
+    , 'Submitted project property information review.'
     , '/projects/assess/properties'
     , 0
```

Comment:
Submitted is a milestone.

Reply:
Same here, the requested change was just to change the name of the step. Was this step also in error?